src/conanfile-h17.py

```diff
@@ -7,7 +7,7 @@ class PalladioConan(ConanFile):
     generators = "cmake"
 
     def requirements(self):
-        self.requires("catch2/2.0.1@bincrafters/stable")
+        self.requires("catch2/2.13.7")
```

Comment:
It seems to work, but I'm wondering why `user/channel` is not necessary? Is this the default for the "conan center" repo?